src/colvarbias_meta.cpp

```diff
@@ -1782,62 +1779,30 @@ int colvarbias_meta::write_replica_state_file()
   if (cvm::debug()) {
     cvm::log("Writing replica state file for bias \""+name+"\"\n");
   }
-  // write down also the restart for the other replicas
-  cvm::backup_file(replica_state_file.c_str());
-  std::ostream *rep_state_os = cvm::proxy->output_stream(replica_state_file);
+
+  std::string const tmp_state_file(replica_state_file+".tmp");
+  std::ostream *rep_state_os = cvm::proxy->output_stream(tmp_state_file);
```

Comment:
Does output_stream() overwrite an existing .tmp file or do we need to call std::remove here in place of backup_file?

Reply:
No, actually it calls `backup_file()` on that file as well. You're right, maybe it'll be cleaner to just `remove()` the `.tmp` file regardless.

Reply:
As long as it's not a fatal error.